.travis.yml

```diff
@@ -24,6 +24,10 @@ matrix:
       env: TARGET=i686-unknown-linux-musl
       rust: stable
       dist: trusty
+    - os: linux
+      env: TARGET=wasm32-unknown-unknow
```

Comment:
Should be "wasm32-unknown-unknown" (missing an 'n' at the end).

Reply:
A copy-paste issue, fixed. :sweat_smile: